lca-ai-stack/source/ui/src/components/call-panel/CallPanel.jsx

```diff
@@ -308,7 +417,22 @@ const TranscriptContent = ({ segment }) => {
   const { settings } = useSettingsContext();
   const regex = settings?.CategoryAlertRegex ?? '.*';
 
-  const { transcript, segmentId, channel } = segment;
+  const {
+    transcript,
+    segmentId,
+    channel,
+    translateClient,
+    translateOn,
+    targetLanguage,
+    agentTranscript,
+  } = segment;
+
+  let r = '';
+  if (translateOn && targetLanguage && targetLanguage !== '') {
+    r = translateTranscript(translateClient, transcript, 'auto', targetLanguage);
```

Comment:
Let's discuss use of 'auto'.. It's convenient, but it incurs an additional cost (from Comprehend detect_dominant_language) for each api call.
Currently LCA knows the source language (it's a stack parameter) so we could use that. 
However we have aspirations to enable automatic language detection in Transcribe streaming at some point.. at which point the stack parameter for language would be a list of possible languages.. then - if we want to avoid 'auto' we'd need to start tagging the transcript segments with the language code detected by Transcribe. 
So I think for now I've argued myself into agreeing that using 'auto' is the best way / most agile forward. Later we could potentially cost optimize by replacing 'auto' with a deterministic source language based on configuration or (someday) Transcribe detected language. 

What do you think @chrislott @babu-srinivasan  ?